UI/GameSettingsScreen.cpp

```diff
@@ -1072,6 +1082,25 @@ UI::EventReturn GameSettingsScreen::OnChangeMacAddress(UI::EventParams &e) {
 	return UI::EVENT_DONE;
 }
 
+UI::EventReturn GameSettingsScreen::OnChangeRemoteISOSubdir(UI::EventParams &e) {
+	//Conform to HTTP standards
+	for (size_t i = 0; i < g_Config.sRemoteISOSubdir.length(); i++)
+	{
+		if (g_Config.sRemoteISOSubdir[i] == ' ') g_Config.sRemoteISOSubdir[i] = '+';
+		if (g_Config.sRemoteISOSubdir[i] == '\\') g_Config.sRemoteISOSubdir[i] = '/';
+		if (g_Config.sRemoteISOSubdir[i] == '?') {
+			g_Config.sRemoteISOSubdir.erase(i); //truncate for safety
```

Comment:
Currently a query string would break the code that creates the game list.  It just combines the subdir with the game filename.  That is why I blocked it.  I would have to strip it off before combining it.  I'll look into it.